src/crewai/crew.py

```diff
@@ -275,46 +275,48 @@ def set_private_attrs(self) -> "Crew":
 
         return self
 
+    def _initialize_default_memories(self):
+        self._long_term_memory = self._long_term_memory or LongTermMemory()
+        self._short_term_memory = self._short_term_memory or ShortTermMemory(
+            crew=self,
+            embedder_config=self.embedder,
+        )
+        self._entity_memory = self.entity_memory or EntityMemory(
+            crew=self, embedder_config=self.embedder
+        )
+        if (
```

Comment:
can we make this a separate method?

like `def _initialize_user_memory(self):`